src/Config.js

```diff
@@ -95,6 +95,15 @@ class ConfigOptions {
     failedDescriptionLabel() { return "M-failed-description"; }
     // allows target branch update in 'guarded_run' mode
     clearedForMergeLabel() { return "M-cleared-for-merge"; }
+
+    approvalUrl() { return "https://github.com/measurement-factory/anubis#voting-and-pr-approvals"; }
+
+    // whether the bot will create 'fake' approval statuses for PR and staged commit.
+    // TODO: make configurable or remove
+    manageApprovalStatus() { return true; }
+
+    // the 'context name' of fake approval status
```

Comment:
s/fake approval/the approval test/. The test can be considered fake (in some sense), but the status is real. I do not think we should use this vague "fakeness" notion in the bot code.